cpp/ql/src/experimental/Security/CWE/CWE-561/FindIncorrectlyUsedSwitch.c

```diff
@@ -12,7 +12,7 @@
 ...
   switch(c1){
 ...
-  dafault: // BAD: maybe it will be right `default`
+  default: // BAD: maybe it will be right `default`
```

Comment:
This correction is probably wrong

Reply:
Yes, please revert. This shows we detect spelling mistakes in C/C++ code 😄

Reply:
personally, I encourage using things like `default0`, `default_`, or `de_fault` since they tend to be functionally equivalent in that they show that your tool catches spelling mistakes, while not actually upsetting spell checkers.

That said, I'm going to drop this and if there's interest, a change like that can be its own distinct PR.

Reply:
It depends on what the query actually checks for. I can have a look at that next week.

Reply:
> I encourage using things like `default0`, `default_`, or `de_fault` since they tend to be functionally equivalent in that they show that your tool catches spelling mistakes

Unfortunately that's not going to work here, as the query looks for specific substitutions. This is the relevant query code:
```codeql
      (
        lb.getName().charAt(0) = "d" or
        lb.getName().charAt(0) = "c"
      ) and
      (
        lb.getName().charAt(1) = "e" or
        lb.getName().charAt(1) = "a"
      ) and
      (
        lb.getName().charAt(2) = "f" or
        lb.getName().charAt(2) = "s"
      )
```
Note that the query is marked as experimental, so the bar on query quality is very low here.